### ISSUE TITLE
Wasm port?

### ISSUE THREAD

**Issue by firsturdiev** [halpz/re3#36]
Web is getting powered with Wasm now and isn't possible to port the game to web? I saw similar thing in cs 1.6. If anyone interested I can ready to contribute

**Comment by kungfooman** [comment]
@firsturdiev Good idea, you need to install emscripten and fix the build process. The game files could just be loaded via Drag&Drop, which would also allow easy modding!

I'm looking forward to see your fork with emscripten support!

**Comment by firsturdiev** [comment]
> [@firsturdiev](https://github.com/firsturdiev) Good idea, you need to install emscripten and fix the build process. The game files could just be loaded via Drag&Drop, which would also allow easy modding!
> 
> I'm looking forward to see your fork with emscripten support!

Thanks, I'll let you know when I started )
